### ISSUE TITLE
Allow forks to create paperclip jar

### ISSUE THREAD

**Issue by Toffikk** [PaperMC/paperweight#19]
The `paperclipJar` task doesnt work on paper forks

![image](https://user-images.githubusercontent.com/62406295/122383216-e9bea300-cf6a-11eb-9856-70b092dd6c17.png)

and it doesn't show up when running `gradlew tasks`

![image](https://user-images.githubusercontent.com/62406295/122383394-15418d80-cf6b-11eb-8005-3f057cf33571.png)

Here is the copy of build.gradle.kts
```gradle
plugins {
    java
    id("com.github.johnrengelman.shadow") version "7.0.0" apply false
    id("io.papermc.paperweight.patcher") version "1.1.0-SNAPSHOT"
}

repositories {
    mavenCentral()
    maven("https://papermc.io/repo/repository/maven-public/") {
        content {
            onlyForConfigurations("paperclip")
        }
    }
    maven("https://maven.quiltmc.org/repository/release/") {
        content {
            onlyForConfigurations("remapper")
        }
    }
}

dependencies {
    remapper("org.quiltmc:tiny-remapper:0.4.1")
    paperclip("io.papermc:paperclip:2.0.0@jar")
}

subprojects {
    apply(plugin = "java")

    java {
        toolchain {
            languageVersion.set(JavaLanguageVersion.of(16))
        }
    }

    tasks.withType<JavaCompile>().configureEach {
        options.isIncremental = true
        options.isFork = true
        options.encoding = "UTF-8"
        options.release.set(16)
    }

    repositories {
        mavenCentral()
        maven("https://libraries.minecraft.net/")
        maven("https://repo.codemc.io/repository/maven-public/")
        maven("https://oss.sonatype.org/content/groups/public/")
        maven("https://papermc.io/repo/repository/maven-public/")
        maven("https://ci.emc.gs/nexus/content/groups/aikar/")
        maven("https://repo.aikar.co/content/groups/aikar")
        maven("https://repo.md-5.net/content/repositories/releases/")
        maven("https://hub.spigotmc.org/nexus/content/groups/public/")
        maven("https://nexus.velocitypowered.com/repository/velocity-artifacts-snapshots/")
        maven("https://oss.sonatype.org/content/repositories/snapshots/")
    }
}
paperweight {
    serverProject.set(project(":Yatopia-Server"))

    usePaperUpstream(providers.gradleProperty("paperCommit")) {
        withPaperPatcher {
            apiPatchDir.set(layout.projectDirectory.dir("patches/api"))
            apiOutputDir.set(layout.projectDirectory.dir("Yatopia-API"))

            serverPatchDir.set(layout.projectDirectory.dir("patches/server"))
            serverOutputDir.set(layout.projectDirectory.dir("Yatopia-Server"))
        }
    }
}
```

**Comment by Proximyst** [comment]
Paste the file in a code block

**Comment by Toffikk** [comment]
done

**Comment by BillyGalbreath** [comment]
`./gradlew build` creates the paperclip jar. To get a non-paperclip jar you have to use `./gradlew shadowJar`

Not very intuitive, but once docs are complete for paperweight it shouldn't be a problem.

**Comment by DenWav** [comment]
I switched that up for Paper core so build is normal and paperclipJar task creates paperclip, forgot to also make that change for forks.

Hopefully that will be more intuitive.

**Comment by DenWav** [comment]
Fixed in 77fe06c637ee4342135eecccaa17846ebce93c44.

**Comment by Toffikk** [comment]
I updated paperweight to 1.1.2 , and the paperclipJar task isn't present , and the build task doesn't generate paperclip
